css-three-column-layout/styles.css

```diff
@@ -0,0 +1,79 @@
+/* stylelint-disable indentation */
+body {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+}
+
+h2,
+ .headers {
+  color: white;
+  text-align: center;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-third {
+  width: calc(100% / 3);
+}
+
+.column-third .images {
+  height: 65px;
+}
+
+.text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: -15px -10px;
```

Comment:
There are applications for it, but generally setting negative padding or margins is not the right way to address issues. Most of the time, when I see negative padding or margin, it is because there is a sibling element you just need to reduce the margin or padding on. See if the element sitting above or below this element in the document flow could have its margin reduced to eliminate the need for this.